src/main/java/com/liferay/faces/util/i18n/internal/I18nImpl.java

```diff
@@ -57,24 +57,21 @@ public I18nImpl() {
 		if (startupFacesContext != null) {
 
 			ExternalContext externalContext = startupFacesContext.getExternalContext();
-			Map<String, Object> applicationMap = externalContext.getApplicationMap();
 			Cache<Locale, ResourceBundle> facesResourceBundleCache;
+			WebConfigParam I18nMaxCacheCapacity = WebConfigParam.I18nMaxCacheCapacity;
+			int maxCacheCapacity = I18nMaxCacheCapacity.getIntegerValue(externalContext);
 
-			String maxCacheCapacityString = externalContext.getInitParameter(I18n.class.getName() +
-					".maxCacheCapacity");
-
-			if (maxCacheCapacityString != null) {
+			if (maxCacheCapacity != I18nMaxCacheCapacity.getDefaultIntegerValue()) {
```

Comment:
@stiemannkj1 Rather than going through the logic of checking for `-1` here, I think that logic an be pushed into the CacheFactoryImpl. In other words, we can always simply just pass whatever the return value of `I18nMaxCacheCapacity.getIntegerValue(externalContext)` is directly into the call to `CacheFactory.getConcurrentLRUCacheInstance(externalContext,	initialCacheCapacity, maxCacheCapacity)`. The convention of `-1` could be documented in the Javadoc in CacheFactory.java